nbconvert/nbconvertapp.py

```diff
@@ -280,6 +281,7 @@ def initialize(self, argv=None):
         """Initialize application, notebooks, writer, and postprocessor"""
         self.init_syspath()
         super(NbConvertApp, self).initialize(argv)
+        ensure_template_dirs_exist()
```

Comment:
Would be better if the template loader was lazy about checking for existance when trying to load templates. Doing this check on each initialization will increase start-up times non-trivially. If this is difficult to implement, we should at least leave a TODO for the proposed changes in 6.0 to change how templates work so it's not a permanent increase to load times.